doc/user-guide/Features-and-supported-standards.md

```diff
@@ -73,7 +73,7 @@
 |0270|[XMPP Advanced Server 2010](http://xmpp.org/extensions/xep-0270.html)||
 |0279|[Server IP Check](http://xmpp.org/extensions/xep-0279.html)|`mod_sic`|
 |0280|[Message Carbons](http://xmpp.org/extensions/xep-0280.html)|`mod_carboncopy`|
-|0313|[Message Archive Management](http://xmpp.org/extensions/attic/xep-0313.html)|`mod_mam`|
+|0313|[Message Archive Management](http://xmpp.org/extensions/attic/xep-0313-0.2.html)|`mod_mam`| <!-- xeps are named like xep-0313-0.1.html, -0.2.html, etc. in attic. xep-tool page refers to 0.2 version -->
```

Comment:
This one should be changed to ordinary link to most recent 0313 version, not the old 0.2 one.

Reply:
done

Reply:
it's mentioned on xep-tool page,
maybe it's better to relaunch xep-tool again? is it work?

Reply:
xep-tool will probably need some improvements first.